r2/r2/templates/subredditstylesheetsource.html

```diff
@@ -33,3 +33,31 @@
 </pre>
 
 ${unsafe(js.use("highlight"))}
+
+%if len(thing.images) > 0:
+  <div id="images">
+    <h2><a name="images">${_("images")}</a></h2>
+    <ul id="image-preview-list" class="image-list">
+      <%def name="make_li(name='', img=None, prototype=False)">
+        <li ${"style='display:none'" if img is None else ""}>
+          <a href="${img}" class="preview">
+            <img id="img-preview-${name}" src="${img}" 
+                 alt="Image ${name}" title="click to preview"/>
+          </a>
+          <div class="description">
+            <b class="img-name">
+              ${name}
+            </b>
+            <br/>
+            <span>link:</span>
+            <pre class="img-url">url(%%${name}%%)</pre>
+            <br/>
+          </div>
+        </li>
+        </%def>
+      ${make_li(prototype=True)}
```

Comment:
the empty `li` is used as a template on the upload form, [here](https://github.com/reddit/reddit/blob/master/r2/r2/templates/subredditstylesheet.html#L137-L151). It doesn't really need to be on this page, but as with the above, I'd rather not duplicate this.  If `SubredditStylesheet` and `SubredditStylesheetSource` are changed to inherit from a base class, then this could be moved into a function in that base template rather than being repeated in both files.